src/components/offer-card/offer-card.jsx

```diff
@@ -23,19 +43,14 @@ const OfferCard = ({className, offers, onCardClick, setActivePinAction, resetAct
             setActivePinAction(offer);
           }}
           onMouseLeave={resetActivePinAction}
-          onClick={() => {
-            onCardClick();
-            getCurrentOfferAction(offer);
-            onOfferClick(offer.id);
-          }}
         >
           {offer.isPremium && className !== PLACES_SUBCLASS ?
             <div className="place-card__mark">
               <span>Premium</span>
             </div>
             : null}
           <div className={`${className}__image-wrapper place-card__image-wrapper`}>
-            <a href="#">
+            <a>
```

Comment:
ссылка без атрибута `href` не валидна, лучше вернуть как было
```suggestion
            <a href="#">
```